docs/index.rst

```diff
@@ -9,13 +9,11 @@ Version 0.1.0
 About
 =====
 
-Gotcha is an API that provides function wrapping, which is the concept
-of injecting a new wrapper function between another function and its
-callsites. Many tools rely function wrapping as a fundamental building
-block. A performance analysis tool may, for example, put function
-wrappers around "read()" and "write()" that trigger a stopwatch around
-the original calls, which allows the tools to measure time an
-application spends in IO.
+Gotcha is an API that provides function wrapping, interposing a wrapper
```

Comment:
Define "wrapper".  It may not be clear to people this is a function.

Otherwise these changes are good.